keyboards/ergodox_ez/keymaps/Bocaj/keymap.c

```diff
@@ -0,0 +1,240 @@
+#include QMK_KEYBOARD_H
+#include "version.h"
+
+#define BASE 0 // default layer
+#define SYMB 1 // symbols
+#define MDIA 2 // media keys
+
+enum custom_keycodes {
+#ifdef ORYX_CONFIGURATOR
+  EPRM = EZ_SAFE_RANGE,
+#else
+  EPRM = SAFE_RANGE,
+#endif
```

Comment:
I don't think you will need this ifdef block.
No need for a custom eeprom initializing keycode, as we have EEP_RST.